src/service/email.js

```diff
@@ -8,13 +8,12 @@ class Email {
 		const transporter = nodemailer.createTransport({
 			host: host,
 			port: port,
-			secure: true,
+			secure: port === '465',
 			auth: {
 				user: username,
 				pass: password
 			}
 		})
-
```

Comment:
Last thing, If you could revert this line I am happy 😄

Reply:
now it is as before